broker-daemon/relayer/relayer-client.spec.js

```diff
@@ -36,6 +37,11 @@ describe('RelayerClient', () => {
   let ENABLE_SSL = true
 
   beforeEach(() => {
+    logger = {
```

Comment:
can this be in another pr? seems unrelated to this change.

Reply:
@treygriffith im not entirely sure where this change came from, but it shouldnt exist in the diff anymore (after updating with master)